server/routes/dataInsights/offenceTypeTab/offenceType.test.ts

```diff
@@ -20,6 +20,12 @@ beforeEach(() => {
       chartEntries: [],
     })
   )
+  chartApiService.getLastModifiedChart.mockResolvedValue(
+    testData.chartLastUpdatedResult({
+      chartName: '3b',
```

Comment:
In `offenceTypeTabPage.ts` we pass `3a` so good to make this `chartName: '3a'` ?